plugins/woocommerce/src/Internal/DataStores/CustomMetaDataStore.php

```diff
@@ -132,7 +133,7 @@ public function add_meta( &$object, $meta ) {
 	 *
 	 * @return bool
 	 */
-	public function update_meta( &$object, $meta ) {
+	public function update_meta( &$object, $meta ) : bool {
```

Comment:
return type is not necessarily `bool`, $wpdb->update return the number of rows, which can theoretically be something other than 0 or 1.

Reply:
Same for delete method.

Reply:
yeah, but `update_meta` returns `1 === $result` where `$result` comes from `$wpdb->update()` ([link](https://github.com/woocommerce/woocommerce/blob/8.0.3/plugins/woocommerce/src/Internal/DataStores/CustomMetaDataStore.php#L159)) and in `delete_meta`, there is a type case to `(bool)` before return ([link](https://github.com/woocommerce/woocommerce/blob/8.0.3/plugins/woocommerce/src/Internal/DataStores/CustomMetaDataStore.php#L94)), so I think it should be accurate.